domaci.css

```diff
@@ -0,0 +1,72 @@
+﻿.container{
+    max-width: 625px;
+    margin: 0 auto;
+}
+.naslov{
+    font-family: 'Merriweather', serif;
+    border-top: 30px solid transparent;
+    color: white;
+    font-weight: 900;
+    margin: 0;
+}
+
+p{
+    color: white;
+    font-size: 15px;
+    font-family: 'Roboto', serif;
+    font-weight: 400;
+    border-bottom: 30px solid transparent;
+   transform: translateY(-4px);
+ 
+}
+
+/*Žuta slika*/
+.zutaslika{
+    background: url("žuta.jpg");
+    background-size: cover;
+    padding-left: 25px;
+    width: 625px;
+    height: 263px;
+   border-bottom: 25px solid white;
```

Comment:
I see you put `border-color: white` because transparent would reveal more of the image. To get around that, you'd need another div tag that would contain the entire div class="zutaslika" and then put the border on that outer div.
You can make the change if you want, otherwise its all good (: